src/csharp/Microsoft.Spark/ML/Param/Param.cs

```diff
@@ -78,4 +75,61 @@ internal Param(JvmObjectReference jvmObject)
         /// <returns>The UID of the parent oject that this <see cref="Param"/> belongs to</returns>
         public string Parent => (string)Reference.Invoke("parent");
     }
+
+    /// <summary>
+    /// A param and its value.
+    /// </summary>
+    public sealed class ParamPair<T> : IJvmObjectReferenceProvider
+    {
+        private static readonly string s_ParamPairClassName = "org.apache.spark.ml.param.ParamPair";
```

Comment:
nit: lets name this s_paramClassName according to the naming convention followed everywhere else

Reply:
Maybe let's have a separate file for `ParamPair`?

Reply:
Isn't the convention of ParamPair is s_ParamPairClassName?